examples/tcpsniffer.cr

```diff
@@ -29,11 +30,15 @@ opts = OptionParser.new do |parser|
   parser.on("-b", "Body printing mode"    ) { bodymode = true }
   parser.on("-x", "Show hexdump output"   ) { hexdump  = true }
   parser.on("-v", "Show verbose output"   ) { verbose  = true }
+  parser.on("-w", "We can easily manage White spaces ( diff(1) uses this as ignore all white space)") { whitespace = true }
```

Comment:
Oh, that message was a comment for you :)
Please write like this for the description.
`"Ignore all packets that contain only white spaces"`